admin/homebrew.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Create a Homebrew recipe for Flocker, using the VERSION environment variable.
+
+Inspired by https://github.com/tdsmith/labmisc/blob/master/mkpydeps.
+"""
+
+from os import environ
+from json import load
+from urllib2 import HTTPError, urlopen
+from hashlib import sha1
+from tl.eggdeps.graph import Graph
+
+
+def get_dependency_graph(application):
+    """
+    Get the dependencies of an application.
+
+    :param unicode application: The name of an application to get the
+        dependencies of.
+
+    :return tl.eggdeps.graph.Graph: Graph of Python dependencies, not
+        including ``application``.
+    """
+    dependency_graph = Graph()
+    dependency_graph.from_specifications([application])
+    dependency_graph.pop(application, None)
```

Comment:
This looks a bit odd. Maybe add a comment explaining what this does, why we're discarding the result and what the `None` argument is for...maybe supply keyword arguments to make it clear.